src/components/AnArticle.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getAnArticle } from "../utils/api";
+
+export const AnArticle = () => {
+  const [article, setArticle] = useState({});
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getAnArticle(article_id).then((articleFromApi) => {
+      setArticle(articleFromApi);
```

Comment:
add a catch block to deal with a non-existent article_id

Reply:
Added